junit-commons/src/main/java/org/junit/gen5/commons/util/ReflectionUtils.java

```diff
@@ -47,12 +48,18 @@ private ReflectionUtils() {
 		return constructor.newInstance();
 	}
 
-	public static Object invokeMethod(Method method, Object testInstance)
+	public static Object invokeMethod(Method method, Object testInstance, Object[] arguments)
```

Comment:
How about using varargs here, i.e. `Object... arguments`. Then you wouldn't have to overload the method.